### PR TITLE
Fix Card image overlay when no `featuredTitle` or `featuredSubtitle`

### LINKED ISSUE #636: Problem with Card style
<Card
			  title={item.title}
			  imageStyle = {{}}
			  image={{uri: 'IMAGE URL HERE'}}>
			 
			</Card>

The picture is darker. It seems like something dark is covering it , the original images source is bright. Once i add it into Card's image , it's darker now

### PR DESCRIPTION
**Pull request by xavier-villelegier**
Fixed #636

### CODE REVIEW

**Comment by iRoachie** [comment]
@xavier-villelegier Rebase from next instead?

**Comment by xavier-villelegier** [comment]
Yep I messed up ahah, done @iRoachie

File: src/card/Card.js [hunk 1]
```diff
@@ -78,29 +78,28 @@ const Card = props => {
...
+                    <Text
+                      style={[
+                        styles.featuredSubtitle,
+                        featuredSubtitleStyle && featuredSubtitleStyle,
+                      ]}
+                    >
+                      {featuredSubtitle}
+                    </Text>}
+                </View>}
+            </Image>
```
Comment (iRoachie) [inline src/card/Card.js]: Supposed to be BackgroundImage

Comment (xavier-villelegier) [reply src/card/Card.js]: Oops the merge went wrong
giphy

**Pull request merged**